k8s/README.md

```diff
@@ -0,0 +1,398 @@
+```
+
+In order to operate the cluster, we'll need the `kubectl` utility, which we can
+install with:
+```
+:; gcloud components install kubectl
+```
+
+Then fetch adminitrative credentials with:
+```
+:; gcloud container clusters get-credentials gobs-cluster
+Fetching cluster endpoint and auth data.
+kubeconfig entry generated for gobs-cluster.
+```
+
+`kubectl` can list the GCE hosts in your cluster:
+```
+:; kubectl get nodes
+NAME                                  STATUS    AGE
+gke-gobs-cluster-03c0208c-node-4ewy   Ready     11m
+gke-gobs-cluster-03c0208c-node-m2g5   Ready     11m
+gke-gobs-cluster-03c0208c-node-n6m9   Ready     11m
+gke-gobs-cluster-03c0208c-node-vr9c   Ready     11m
+gke-gobs-cluster-03c0208c-node-w1k6   Ready     11m
+```
+
+And, while you shouldn't have to do so, you can get a terminal on any of these hosts with i.e.
+```
+:; gcloud compute ssh gke-gobs-cluster-03c0208c-node-4ewy
+Updated [https://www.googleapis.com/compute/v1/projects/gobs-project].
+Warning: Permanently added '104.197.228.216' (RSA) to the list of known hosts.
+Warning: Permanently added '104.197.228.216' (RSA) to the list of known hosts.
+Linux gke-gobs-cluster-03c0208c-node-4ewy 3.16.0-4-amd64 #1 SMP Debian 3.16.7-ckt20-1+deb8u4google (2016-01-26) x86_64
+
+Welcome to Kubernetes v1.2.0!
+
+You can find documentation for Kubernetes at:
+  http://docs.kubernetes.io/
+
+You can download the build image for this release at:
+  https://storage.googleapis.com/kubernetes-release/release/v1.2.0/kubernetes-src.tar.gz
+
+It is based on the Kubernetes source at:
+  https://github.com/kubernetes/kubernetes/tree/v1.2.0
+
+For Kubernetes copyright and licensing information, see:
+  /usr/local/share/doc/kubernetes/LICENSES
+
+ver@gke-gobs-cluster-03c0208c-node-4ewy:~$ 
+```
+
+Since we haven't deployed any applications yet, there are no Pods yet:
+```
+:; kubectl get pods
+:;
+```
+
+At this point your kubernetes cluster is at a point where we can
+deploy an application!
+
+## Gleaming the Kube ##
+
+Before we start running's Gob's app, let's start by setting up _namerd_.
+
+### `namerd` ###
+
+The _namerd_ service isn't required to run _linkerd_, but it allows us to
+manage routing rules centrally so that they may be updated without redeploying
+linkerd.
+
+Before deploying namerd, install the `namectl` utility which lets us interact
+with the namerd API from the commandline. If you have [go](https://golang.org)
+configured on your system, this is as simple as:
+```
+:; go install github.com/buoyantio/namectl
+:; namectl              
+namectl controls the namerd rpc route management service.
+
+Usage:
+  namectl [flags]
+  namectl [command]
+
+Available Commands:
+  list		List all Dtab names.
+  get		Get a Dtab by name.
+  new		Create a new Dtab.
+  update	Update an existing Dtab.
+
+error: command required
+```
+
+The [`k8s/namerd`](./namerd) contains all of the configuration files needed to
```

Comment:
Missing a word here. I think it should be: "The k8s/namerd directory contains...".